src/main/java/ch/njol/skript/lang/Expression.java

```diff
@@ -53,7 +53,7 @@
 	/**
 	 * Get the single value of this expression.
 	 * <p>
-	 * This method may only return null if it always returns null for the given event, i.e. it is equivalent to getting a random element out of {@link #getAll(Event)} or null iff
+	 * This method may only return null if it always returns null for the given event, i.e. it is equivalent to getting a random element out of {@link #getAll(Event)} or null if
```

Comment:
I think this one's intentional, but not sure if it's actual English
![image](https://user-images.githubusercontent.com/29547183/178677136-6f914411-a43a-4b1f-9f8a-2588c0f28d8d.png)